enterprise/frontend/src/metabase-enterprise/embedding/components/EmbeddingAppSameSiteCookieDescription/EmbeddingAppSameSiteCookieDescription.tsx

```diff
@@ -0,0 +1,72 @@
+import { jt, t } from "ttag";
+import { useSelector } from "metabase/lib/redux";
+import { alpha, color } from "metabase/lib/colors";
+import { getDocsUrl, getSetting } from "metabase/selectors/settings";
+import { Center, Stack, Text } from "metabase/ui";
+import ExternalLink from "metabase/core/components/ExternalLink";
+import { isEmpty } from "metabase/lib/utils";
+import { isSameOrigin } from "metabase/lib/dom";
+
+export const EmbeddingAppSameSiteCookieDescription = () => {
+  const docsUrl = useSelector(state =>
+    getDocsUrl(state, {
+      page: "embedding/interactive-embedding#embedding-metabase-in-a-different-domain",
+    }),
+  );
+
+  const embeddingSameSiteCookieSetting = useSelector(state =>
+    getSetting(state, "session-cookie-samesite"),
+  );
+  const embeddingAuthorizedOrigins = useSelector(state =>
+    getSetting(state, "embedding-app-origin"),
+  );
+
+  const shouldDisplayNote =
+    embeddingSameSiteCookieSetting !== "none" &&
+    authorizedOriginsContainsNonInstanceDomain(embeddingAuthorizedOrigins);
+
+  return (
+    <Stack mb="1rem" spacing="sm">
+      <Text fw="bold">{t`SameSite cookie setting`}</Text>
+      {shouldDisplayNote && <AuthorizedOriginsNote />}
+      <Text>{t`Determines whether or not cookies are allowed to be sent on cross-site requests. You’ll likely need to change this to None if your embedding application is hosted under a different domain than Metabase. Otherwise, leave it set to Lax, as it's more secure.`}</Text>
+      <Text>{jt`If you set this to None, you'll have to use HTTPS (unless you're just embedding locally), or browsers will reject the request. ${(
+        <ExternalLink key="learn-more" href={docsUrl}>
+          Learn more
+        </ExternalLink>
+      )}`}</Text>
+    </Stack>
+  );
+};
+
+function AuthorizedOriginsNote() {
+  return (
+    <Center
+      data-testid="authorized-origins-note"
+      bg={alpha("warning", 0.1)}
+      w="22rem"
+      p="0.75rem"
+      style={{
```

Comment:
I think we should still be using emotion instead of inline styles when we need to add anything custom that is not supported by the style props

Reply:
The component looks pretty similar to `Alert`, can we use it?

Reply:
I will use emotion instead of inline styles going forward :pray: 

And thanks for surfacing the `Alert` component.
I thought it would be under the name `Warning`, so I did not end up finding one that was similar.

Thanks Sasha! :heart: